src/main/java/com/xebialabs/overthere/util/WindowsCommandLineArgsSanitizer.java

```diff
@@ -24,9 +24,18 @@
 
 public class WindowsCommandLineArgsSanitizer {
 
-    private static final char[] SPECIAL_CHARS = "|<>&^\r\n".toCharArray();
+    private static final char[] SPECIAL_CHARS = " |<>&^\r\n".toCharArray();
```

Comment:
sorry I moved in too fast , this only is needed to determine if the Argument to be escaped has any special characters or not , space is never escaped by a caret in the sanitize method